.github/workflows/build-custom.yml

```diff
@@ -79,6 +73,10 @@ jobs:
   build-chainlink:
     name: Build Chainlink Image
     runs-on: ubuntu-latest
+    environment: build-develop
```

Comment:
@CL-Andrew may need you to confirm that the environment can be used here and that it will work with the various refs and branches that get passed in from external repos (`chainlink-solana`, `chainlink-starknet`, etc. passing in `cl_ref`.